versions/3.0.md

```diff
@@ -128,8 +128,6 @@ Field Name | Type | Description
 <a name="oasVersion"></a>openapi | `string` | **Required.** Specifies the OpenAPI Specification version being used. It can be used by tooling Specifications and clients to interpret the version. The structure shall be `major`.`minor`.`patch`, where `patch` versions _must_ be compatible with the existing `major`.`minor` tooling. Typically patch versions will be introduced to address errors in the documentation, and tooling should typically be compatible with the corresponding `major`.`minor` (3.0.*). Patch versions will correspond to patches of this document.
 <a name="oasInfo"></a>info | [Info Object](#infoObject) | **Required.** Provides metadata about the API. The metadata can be used by the clients if needed.
 <a name="oasHosts"></a>hosts | [Hosts Object](#hostsObject) | An array of Host objects which provide `scheme`, `host`, `port`, and `basePath` in an associative manner.
-<a name="oasConsumes"></a>consumes | [`string`] | A list of media types the APIs can consume. This is global to all APIs but can be overridden on specific API calls. Value MUST be as described under [Media Types](#mediaTypes).
```

Comment:
For incompatible changes to OAS from 2.0 to 3.0, we need a guide (separate from the spec) on how to convert (or interpret). I.e. if `consumes`/`produces` are removed here, how does one transform a 2.0 document to 3.0 and preserve the same API definition? This also applies to removing  `consumes`/`produces` on an operation. It's only slightly clearer there, since the `representations`  exist, but there are still some ambiguities: if produces lists more than one media type, which one is mapped to the 2xx response bodies, and what happens to the others?  Example 2: how are the top level  `consumes`/`produces` merged with the operation  `consumes`/`produces` to yield the same effective `representations` in request and response bodies?